packages/node/src/models/state-channel.ts

```diff
@@ -212,6 +212,23 @@ export class StateChannel {
     return topLevelKey;
   }
 
+  // useful for debugging
+  public prettyPrintFB() {
+    const balances = deserializeFreeBalanceState(this.freeBalanceAppInstance!
+      .state as FreeBalanceStateJSON).balancesIndexedByToken;
+
+    const ret = {} as any;
+
+    for (const tokenAddress of Object.keys(balances)) {
+      const ret2 = {} as any;
+      for (const coinTransfer of balances[tokenAddress]) {
+        ret2[coinTransfer.to] = coinTransfer.amount;
+      }
+      ret[tokenAddress] = ret2;
+    }
```

Comment:
I think this function is doing a similar transform for something we have a utility function for within `free-balance.ts`.

Reply:
it's similar but not the same